charts/core/templates/csp-deployment.yaml

```diff
@@ -26,6 +26,10 @@ spec:
       imagePullSecrets:
         - name: {{ .Values.awsbilling.imagePullSecrets }}
       {{- end }}
+      {{- if .Values.global.azurebilling.imagePullSecrets }}
+      imagePullSecrets:
+        - name: {{ .Values.global.azurebilling.imagePullSecrets }}
+      {{- end }}
       containers:
```

Comment:
If we do not want to maintain a seperate fork or branch for building the CNAB for Azure, we need to follow these guidelines. https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux#update-the-helm-chart

we need to add in some extra env variables here or using somethibg like

        # envFrom:
        #   - configMapRef:
        #       name: neuvector-csp-pod
        env: